test/utils/string.formatter.spec.unit.ts

```diff
@@ -30,11 +29,19 @@ describe("String formatter tests", () => {
     expect(formattedString).toEqual("hello. This string is dodgy ! @ £$% &*()€#     ");
   });
 
-  it("should not remove printable chars", () => {
-    const preFormattedString: string = "hello. This string is not dodgy !@£$%&*{}()€#";
+  it("should not remove printable chars in sentence", () => {
+    const preFormattedString: string = "hello. This string is not dodgy, !@£$%&*{}()€#";
     const formattedString: string = removeNonPrintableChars(preFormattedString);
 
-    expect(formattedString).toEqual("hello. This string is not dodgy !@£$%&*{}()€#");
+    expect(formattedString).toEqual("hello. This string is not dodgy, !@£$%&*{}()€#");
+  });
+
+  it("should not remove any printable chars when entire set is provided", () => {
+    const preFormattedString: string = "ÀÁÂÄÈÉÊËÌÍÎÏÒÓÔÖÙÚÛÜẀẂŴẄỲÝŶŸàáâäèéêëìíîïòóôöùúûüẁẃŵẅỳýŷÿABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789&@£$€¥.,:;–-‘’'()[]{}<>!“”»«\"?/\\*=#%+ \r\n";
+    const formattedString: string = removeNonPrintableChars(preFormattedString);
+    const test: boolean = preFormattedString === formattedString;
```

Comment:
looks like you're testing the same thing twice here. if toEqual isn't enough you can use toStrictEqual

Reply:
They are different but will change.